modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/SqlSystemViewsSelfTest.java

```diff
@@ -1567,7 +1567,6 @@ public void testCachesViews() throws Exception {
             "ON C.CACHE_NAME = CG.CACHE_GROUP_NAME WHERE C.CACHE_NAME like 'cache%' " +
             "AND C.CACHE_MODE = CG.CACHE_MODE " +
             "AND C.ATOMICITY_MODE = CG.ATOMICITY_MODE " +
-            "AND COALESCE(C.AFFINITY, '-') = COALESCE(CG.AFFINITY, '-') " +
```

Comment:
What about to fix Affinity.toString instead?

Reply:
thought about it in the first commit
(https://github.com/apache/ignite/pull/9215/commits/94ab8aa0f345acb5901e9e2bf8abf98849894101), 
but this change looks unexpected and better in a separate ticket

Reply:
I think that fix of affinity.toString will fix this issue too. And toString is a core problem, so let's fix it instead, if it possible.

Reply:
AFAIK - `lastWakeUpTime` is changes between to consequent `toString` invocation.
I don't think we should remove it from `Affinity#toString`.

Reply:
Sorry, I don't found lastWakeUpTime in affinity functions, could you please provide a link? I see that AffinityFunction has static fields only, like  `partitions`, `backups`.

The problem I saw that RendevouzAffinityFunction has Ignite instance in toString. And we return it to user when it ask for affintiy function from view. Looks incorrect, isn't it?